app/code/Magento/CustomerGraphQl/Model/Customer/GetCustomer.php

```diff
@@ -70,13 +70,15 @@ public function execute(ContextInterface $context): CustomerInterface
 
         try {
             $customer = $this->customerRepository->getById($currentUserId);
+            // @codeCoverageIgnoreStart
         } catch (NoSuchEntityException $e) {
             throw new GraphQlNoSuchEntityException(
                 __('Customer with id "%customer_id" does not exist.', ['customer_id' => $currentUserId]),
                 $e
             );
         } catch (LocalizedException $e) {
             throw new GraphQlInputException(__($e->getMessage()));
+            // @codeCoverageIgnoreEnd
```

Comment:
Generated report after using `@codeCoverageIgnoreStart`
![image](https://user-images.githubusercontent.com/11827230/66473883-d6bab680-ea98-11e9-8346-6de42d1290c3.png)

@lenaorobei, should I also specify the reason there?
In most cases, I would like to place `@codeCoverageIgnoreStart - @codeCoverageIgnoreEnd` in those parts of code where we try to catch errors related to 'Unique constraint violation' and similar database problems. It can be a temporary decision while we haven't reached full coverage. After that, it is easy to locate those parts of code and introduce mocks for the emulation of the database issues.